test/functional/api/extmark_spec.lua

```diff
@@ -834,6 +834,38 @@ describe('API/extmarks', function()
     check_undo_redo(ns, marks[1], 0, 4, 0, 6)
   end)
 
+  it("tabs don't write spaces when deleted with inline virtual text", function()
+    -- ins_bs in edit.c
+    feed(':set shiftwidth=2<cr><esc>')
+    feed(':set softtabstop=2<cr><esc>')
+    feed(':set tabstop=8<cr><esc>')
+    feed(':%d<cr>')
+    feed('0i<tab><esc>')
```

Comment:
This doesn't insert a TAB. It inserts two spaces because 'shiftwidth' and 'smarttab' are set.

Reply:
> When on, a \<Tab\> in front of a line inserts blanks according to
	'shiftwidth'.
	
Are 'blanks' spaces?